src/lib/__tests__/__handCheckTests__/handCheck.test.js

```diff
@@ -1,84 +1,84 @@
 const { handCheck } = require("lib/handCheck");
 
 describe("handCheck function", () => {
-  test("should return 10", () => {
+  test("should return 10, Royal Flush", () => {
     //GIVEN
     const hand = ["C14", "C13", "C12", "C11", "C10"];
     //WHEN
     const result = handCheck(hand);
     //THEN
-    expect(result).toBe(10);
+    expect(result).toStrictEqual([10, "Royal Flush"]);
```

Comment:
How come you decided to return an array of info. As long as the data can help you derive what is the name of the winning hand a helper function could do that for you. 10 and "Royal Flush" refer to the same meaning. So 10 is enought and a function that takes 10 and returns "Royal Flush" would be sufficient. This would make you more versatile in any translations needed in the future.

Reply:
Number and string used to be defined in two separate functions that worked the same way. So, i decided to merge them into one.

Reply:
ok!